resources/warnings.json

```
@@ -16863,7 +16863,7 @@
     "versions": [
       {
         "lastVersion": "5.v77a_37f62782d",
-        "pattern": ".*"
+        "pattern": "(1|5)[.].+)"
```

Unbalanced parentheses.
```suggestion
        "pattern": "([15])(|[.-].+)"
```
(More than needed, but consistent with most other patterns.)